.travis.yml

```diff
@@ -1,17 +1,33 @@
-sudo: false
-cache: pip
-dist: trusty
+group: travis_latest
 language: python
-# For pypy versions, see https://github.com/pyenv/pyenv/tree/master/plugins/python-build/share/python-build
-python:
-  - 3.6
-  - 3.5
-  - 3.4
-  - 2.7
-  - pypy3.5-5.9.0
-  - pypy2.7-5.8.0
+cache: pip
+jobs:
+    include:
+      # We put 3.6 first since this is where flake8 and docs are testted
+      - python: 3.6
+      # We put pypy towards the beginning because for some reason the build takes much longer
+      - python: pypy
+      - python: pypy3
+      - python: 2.7
+      - python: 3.4
+      - python: 3.5
+      - python: 3.7
+        dist: xenial
```

Comment:
Please add a comment on the xenial and sudo things, since I assume at some point they won't be needed.

Reply:
Added comment